org.opent2t.sample.multisensor.superpopular/test/multisensorTests.js

```diff
@@ -235,15 +235,19 @@ function runMultisensorTests(settings) {
             return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesLastchanged', [deviceIds['opent2t.d.sensor.motion']])
                 .then((response) => {
                     t.is(response.rt[0], 'opent2t.r.timestamp');
-                    let testChangedAt = testSettings.inputLastReading["motion_changed_at"];
-                    let testUpdatedAt = testSettings.inputLastReading["motion_updated_at"];
-                    if ((!testChangedAt || isNaN(testChangedAt))
-                        && (!testUpdatedAt || isNaN(testUpdatedAt))) { 
-                        t.true(response.timestamp === undefined, "Expected undefined value for motion-lastchanged");
-                    }
-                    else {
+                    if(testSettings.inputLastReading !== undefined) { //Wink-Only
```

Comment:
> //Wink-Only [](start = 69, length = 12)

we should not have an if for a provider, all of our responses should be the same.  Is this a wink bug or a smartthings bug?